model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationRequestV1.java

```diff
@@ -15,21 +15,19 @@
  *******************************************************************************/
 package com.ge.predix.acs.rest;
 
-import java.util.Set;
-import java.util.LinkedHashSet;
-
-import org.apache.commons.lang.builder.EqualsBuilder;
-import org.apache.commons.lang.builder.HashCodeBuilder;
-
 import com.ge.predix.acs.model.Attribute;
-
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
 
 @SuppressWarnings({ "javadoc", "nls" })
 @ApiModel(description = "Policy evaluation request for V1.")
 public class PolicyEvaluationRequestV1 {
-    public static final LinkedHashSet<String> EMPTY_POLICY_EVALUATION_ORDER = new LinkedHashSet<String>();
+    private static final LinkedHashSet<String> EMPTY_POLICY_EVALUATION_ORDER = new LinkedHashSet<String>();
```

Comment:
we should revert this change since i'm removing LinkedHashSet in favor of the Set interface in another feature branch